p2.py

```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
+#start
 #https://leetcode.com/problems/palindrome-number/
+=======
+#https://leetcode.com/problems/palindrome-number
+>>>>>>> 049aac22aac08949e056adbe56d1cb945f762627
 #clink the above link to the problem description
 class Solution:
     def isPalindrome(self, x: 'int') -> 'bool':
```

Comment:
1. Remove unnecessary integer conversion. 
2. Use "//" for integer division in Python3
3. Try to do this problem without converting integer to string